server/oauth-metadata.go

```diff
@@ -211,7 +211,7 @@ func (s *IDPServer) serveJWKS(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
 	sk, err := s.oidcPrivateKey()
 	if err != nil {
-		writeJSONError(w, http.StatusInternalServerError, "server_error", "internal server error")
+		writeHTTPError(w, r, http.StatusInternalServerError, "server_error", "internal server error")
```

Comment:
meta-comment: here and in quite a few other places we don't log the actual `error` value. I imagine that value would be quite useful for debugging. consider logging all `error` values internally (without returning them in response bodies)